app/audit/events/QuestionnaireAuditEvent.scala

```diff
@@ -19,10 +19,11 @@ package audit.events
 import models.QuestionnaireModel
 import play.api.libs.json.{JsObject, Json}
 import play.api.mvc.{AnyContent, Request}
+import services.RegistrationAuditEvent
 import uk.gov.hmrc.http.HeaderCarrier
 
 class QuestionnaireAuditEvent(questionnaireModel: QuestionnaireModel)(implicit hc:HeaderCarrier, request:Request[AnyContent])
-  extends RegistrationAuditEvent(
+  extends RegistrationAuditEvent (
     auditType = "Questionnaire",
       detail = Json.toJson(questionnaireModel).as[JsObject]
   )(hc,request)
```

Comment:
This won't be needed anymore - as the `QuestionnaireModel` should be passed directly to `AuditService.sendEvent[T]`

Meaning, this intermediary class that used to be required is now redundant